custom-commands/tinker/web/tinker

```diff
@@ -3,7 +3,7 @@
 ## Description: Run project PHP REPL. Arguments are accepted in Laravel projects (with caveats)
 ## Usage: tinker
 ## Example: "ddev tinker"
-## ProjectTypes: laravel,drupal7,drupal8,drupal9
+## ProjectTypes: laravel,drupal7,drupal8,drupal9,drupal10
```

Comment:
You'll want to add ExecRaw I think - should solve your exec problems.

```suggestion
## ProjectTypes: laravel,drupal7,drupal8,drupal9,drupal10
## ExecRaw: true
```

Reply:
Added

Reply:
Just so you know, you could have just clicked "commit suggestion"